pkgs/top-level/all-packages.nix

```diff
@@ -34933,7 +34933,12 @@ with pkgs;
     inherit (darwin) sigtool;
   };
 
-  qemu-utils = callPackage ../applications/virtualization/qemu/utils.nix { };
+  qemu-utils = callPackage ../applications/virtualization/qemu/utils.nix {
+    qemu = qemu.override (o: {
+      toolsOnly = true;
+      hostCpuTargets = [ ];
```

Comment:
Shouldn't this be implied by `toolsOnly`?

Reply:
Done.